lib-php/website/reverse-only-search.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+require_once(CONST_LibDir.'/ParameterParser.php');
+
+$oParams = new Nominatim\ParameterParser();
+$sOutputFormat = $oParams->getSet('format', array('text', 'json'), 'text');
```

Comment:
This should support the same output formats as normal search. Otherwise people get confusing ' wrong format'  responses even though that is not the problem.

Reply:
Sure, Done